Module/client/src/components/app.jsx

```diff
@@ -17,25 +18,49 @@ class App extends React.Component {
     this.getData();
   }
 
+  // initial and only request to server for mockData
   getData() {
     axios.get('/listinginfo').then((response) => {
       let { id } = this.state;
-      if (!id || (id > 100) || (id < 0)) { id = 0; }
+      // sets id to the first mock data example if id is undefined or outside range
+      if (!id || (id > 101) || (id < 0)) { id = 0; }
```

Comment:
I get the idea behind doing this, but wouldn't it make more sense for it to show a 404 page not found?

Reply:
Yeah when in production i'll make it send a 404, but for now sometimes i forget to type in the id number. Thanks!